irods/start_provider.sh

```diff
@@ -23,7 +23,10 @@ python configure_update_collection_mtime_plugin.py
 python configure_unified_storage_tiering_plugin.py
 python configure_users.py
 pkill irodsServer
+pkill irodsReServer
```

Comment:
is this necessary?    was it merely overlooked?   doesn't the irodsServer process take care of its children?

Reply:
Ah yes, left in from debugging. I shall remove